config/default.yaml

```
@@ -165,6 +165,17 @@ auth:
       clientId: ~
       discoveryUrl: ~
 
+    exampleProvider:
+      displayName: "Example Provider"
+      enabled: false
+      # Disable userInfo request for providers such as ADFS
+      fetchUserInfo: false
+      claimsFallback:
+        email: []
+        name: []
```

Since we are only allowing a single one (that's fine) this should be a string, not an array.
```suggestion
        email: upn
        # name: []
```